command/cli.go

```diff
@@ -67,7 +67,8 @@ type MetaArgs struct {
 	Vars         map[string]string
 	VarFiles     []string
 	// set to "hcl2" to force hcl2 mode
-	ConfigType configType
+	ConfigType       configType
+	StrictValidation bool
```

Comment:
This is faithful to what's in the code at the moment, but I think `Strict` is a bit vague with respect to what the command-line option implies.
Do you think we can consider renaming this? Maybe something explicit like `WarnOnUndeclaredVar`? This could also be replicated in `ValidationOptions`, since the current `Strict` seems only to be used for vars, and prior to this commit, is not settable via command-line options.
I'm worried that in the future one of us unknowing what the implications are might see `Strict`, apply that for another unrelated behaviour that deserves some varying level of strictness when validating, and end-up producing some error to a user that doesn't expect that.

Reply:
Yes, I understand completely and agreed. I will update the name to avoid future confusion.